src/clientsettingsdlgbase.ui

```diff
@@ -63,12 +63,12 @@
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
-              <enum>QSizePolicy::Preferred</enum>
+              <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
-               <height>13</height>
+               <height>10</height>
```

Comment:
These changes appear to be layout-relevant? Could they be the reason for the observed changes?

Reply:
Yep, that's now what I'm thinking - not sure why I didn't spot your comment earlier!